example/test/function_component_test.dart

```diff
@@ -335,7 +335,7 @@ class FancyInputApi {
   FancyInputApi(this.focus);
 }
 
-final FancyInput = react.forwardRef((props, ref) {
+final FancyInput = react.forwardRef2((props, ref) {
   final inputRef = useRef();
 
   useImperativeHandle(
```

Comment:
I'm getting a runtime error when serving this example. It looks like this `useImperativeHandle` is expecting a `Ref` object, but getting a `JsRef`?
![Screen Shot 2020-09-08 at 11 15 32 AM](https://user-images.githubusercontent.com/51122966/92513116-a6a5f700-f1c4-11ea-9470-03d14e39dfc4.png)

Reply:
Interesting... good catch, I'll look into this!

Reply:
Addressed in be896a7844a2f13e1a0a987266f9822b1fab965d!